src/components/cargo/CargoManager.ts

```diff
@@ -113,18 +115,19 @@ export class CargoManager {
 
     public registerCommandHelpingCreateProject(commandName: string, isBin: boolean): Disposable {
         return commands.registerCommand(commandName, () => {
-            const cwd = workspace.rootPath;
-            if (!cwd) {
-                window.showErrorMessage('Current document not in the workspace');
-                return;
-            }
-            const projectType = isBin ? 'executable' : 'library';
-            const placeHolder = `Enter ${projectType} project name`;
-            window.showInputBox({ placeHolder: placeHolder }).then((name: string) => {
-                if (!name || name.length === 0) {
+            this._currentWorkingDirectoryManager.cwd().then(cwd => {
+                if (!cwd) {
+                    window.showErrorMessage('Current document not in the workspace');
```

Comment:
I think it should be "Current document is not in the workspace", shouldn't it?

Reply:
I believe that it was already written this way. Apart from refactoring that part I did not change the string's content. But now that you have mentioned it, I just amended it. 😃